pkg/ccip/bindings/offramp/offramp.go

```diff
@@ -178,6 +176,64 @@ type UpdateDeployables struct {
 
 // Config types that implements getter fetching interface with rpc client
 
+// OCR3Base represents the OCR3 base configuration stored on-chain
+type OCR3Base struct {
+	ChainID uint8       `tlb:"## 8"`
+	Commit  *OCR3Config `tlb:"maybe ^"`
+	Execute *OCR3Config `tlb:"maybe ^"`
+}
+
+func (c *OCR3Base) FetchResult(ctx context.Context, client ton.APIClientWrapped, block *ton.BlockIDExt, contractAddr *address.Address, _ []interface{}) error {
+	return ccipcommon.FetchResultHelper(ctx, client, block, contractAddr, ocr3BaseGetter, nil, c)
+}
```

Comment:
nit: We have sprinkled many of these functions across bindings, they are all the same: https://github.com/search?q=repo%3Asmartcontractkit%2Fchainlink-ton+FetchResult%28ctx+context.Context&type=code

imo we should revisit this `FetchResult` interface and `ccipcommon.FetchResultHelper` util and compose it back more fluently